src/src/com/microsoft/aad/adal/DefaultTokenCacheStore.java

```diff
@@ -367,4 +370,78 @@ public boolean contains(String key) {
 
         return mPrefs.contains(key);
     }
+    
+    private TokenCacheItem getFamilyRefreshTokenItemForUser(String userId) {       
+        
+        Iterator<TokenCacheItem> results = this.getTokensForUser(userId).iterator();
+        TokenCacheItem tokenItem = new TokenCacheItem();   
+        while (results.hasNext()) {
+            final TokenCacheItem tokenCacheItem = results.next();
+            if(tokenCacheItem.getUserInfo() != null
+                    && !StringExtensions.IsNullOrBlank(tokenCacheItem.getFamilyClientId())
+                    && tokenCacheItem.getFamilyClientId().equalsIgnoreCase("1")){
```

Comment:
I don't understand the comparision for 1 here? What if we have third part family support?